src/app/core/views/portinfo/digital/index.ts

```diff
@@ -45,4 +45,26 @@ export const DigitalPortInfo: PortInfoRecord<DigitalComponent> = {
             "outputs": [CalcPortPos(V(0.5, 0), V(1, 0))], // 1 output
         })),
     },
+    "XORGate": {
+        ...DefaultDigitalPortInfo,
+        AllowChanges: true,
+        ChangeGroup:  "inputs",
+
+        // Generate configs for 2->8 input ports
+        PositionConfigs: [2,3,4,5,6,7,8].map((numInputs) => ({
+            "inputs":  CalcQuadCurvePortPositions(numInputs, 1.2, .48, V(-1, 0)),
+            "outputs": [CalcPortPos(V(.58, 0), V(1, 0))], // 1 output
+        })),
+    },
+    "XNORGate": {
+        ...DefaultDigitalPortInfo,
+        AllowChanges: true,
+        ChangeGroup:  "inputs",
+
+        // Generate configs for 2->8 input ports
+        PositionConfigs: [2,3,4,5,6,7,8].map((numInputs) => ({
+            "inputs":  CalcQuadCurvePortPositions(numInputs, 1.2, .48, V(-1, 0)),
+            "outputs": [CalcPortPos(V(.8, 0), V(.7, 0))], // 1 output
```

Comment:
I think the `outputs` here can be the same as `XORGate`, the circle for the `not` can simply lay on-top of the port
![Screen Shot 2022-11-08 at 1 51 01 PM](https://user-images.githubusercontent.com/29997057/200650472-2680e4d3-c154-48d6-94f7-c0324242244b.png)
(this is a screenshot from master)